docs/integration.rst

```diff
@@ -701,6 +701,77 @@ Transfer Service
 They also use :class:`airflow.contrib.hooks.gcp_transfer_hook.GCPTransferServiceHook` to communicate with Google Cloud Platform.
 
 
+Cloud Vision
+''''''''''''
+
+Cloud Vision Product Search Operators
```

Comment:
Please rebase.

Reply:
What do you mean? I've rebased on top of `apache/master` and there were no conflicts.

Reply:
I've updated the `integration.rst` to the new format (only linking to `code.rst`).